src/qz/printer/status/PrinterStatus.java

```diff
@@ -54,7 +59,7 @@ public static PrinterStatus getFromCupsString(String reason, String issuingPrint
     }
 
     public String toString() {
-        String returnString = type.getName() + ": Level " + type.getSeverity() + ", StatusCode " + type.getCode() + ", From " + issuingPrinterName;
+        String returnString = type.getName() + ": Level " + type.getSeverity() + ", StatusCode " + type.getCode() + ", From " + issuingPrinterDescription;
```

Comment:
To avoid confusion please keep `issuingPrinterName` in the logs.  If description is desired, add it as an additional field.

Reply:
The reason this changed is because this is the status' toString, and I wanted it to reflect exactly what the status will report to the client. Should I still go ahead with this change?